<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teacher_Window</class>
 <widget class="QDialog" name="Teacher_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teacher</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FiraCode Nerd Font Mono Ret</family>
    </font>
   </property>
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 255, 0);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 0);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FiraCode Nerd Font Mono Ret</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 61, 64);
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Teacher</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="type">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Info:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_teacher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="name_teacher">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="description_teacher">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(33, 75, 57);
color:white;
border-style:outset;
border-radius:9px;
border-width:2px;
border-color:black;</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
